client/src/components/ReviewWizard/Form.tsx

```diff
@@ -43,133 +43,141 @@ export default function Form({ initialReview, onChange, onSubmit }: FormProps):
     reValidateMode: 'onBlur',
   });
 
-  const convertAndSave = () => onChange(convertFormValuesToReview(getValues()));
+  const [lastSavedValues, setLastSavedValues] = useState(getValues());
+
+  const convertAndSave = () => {
+    // only save the draft if the new values are different from the old values
+    const currentValues = getValues();
+    if (!_isEqual(currentValues, lastSavedValues)) {
+      onChange(convertFormValuesToReview(getValues()));
+      setLastSavedValues(currentValues);
+    }
+  };
+
   const convertAndSubmit = (formReview: FormReview) => onSubmit(convertFormValuesToReview(formReview));
 
   const reviewItemColSpan = { lg: 12, sm: 24 };
 
   return (
     <form>
-      <DraftsContext.Provider value={convertAndSave}>
```

Comment:
the only change here is that I removed the unused `DraftsContextProvider`, the rest is just indented in, so Git marks it as changed. I promise I didn't touch it!